src/components/CurriculumComponents/CurriculumVisualiser/CurriculumVisualiser.tsx

```diff
@@ -351,14 +352,15 @@ const CurriculumVisualiser: FC<CurriculumVisualiserProps> = ({
                   data-testid="unit-cards"
                 >
                   {units
-                    .filter((unit: Unit) =>
-                      isVisibleUnit(
+                    .filter((unit: Unit) => {
+                      const isVisible = isVisibleUnit(
```

Comment:
**nitpick**: no need to change this now, but it's good practice to not have non-functional changes in PRs, as it makes git history harder to read

Really not a big issue :)